libs/model/src/models/notification.ts

```diff
@@ -36,8 +36,12 @@ export default (
     'Notification',
     {
       id: { type: dataTypes.INTEGER, primaryKey: true, autoIncrement: true },
-      notification_data: { type: dataTypes.TEXT, allowNull: false },
-      chain_event_id: { type: dataTypes.INTEGER, allowNull: true },
+      notification_data: { type: dataTypes.TEXT, allowNull: true },
+      chain_event_id: {
```

Comment:
We can probably drop this column cc @timolegros

Reply:
I'll ticket this now.

Reply:
Yep we can remove this `chain_event_id` now that old notifications that had it defined have been cleaned up. That said you probably don't want to do that in this PR since it will require client changes.